client/app/assets/less/inc/base.less

```diff
@@ -222,6 +222,14 @@ text.slicetext {
     }
 }
 
+.fa-external-link {
+    font-size: inherit;
```

Comment:
Why is this needed? As far as I remember, FA already uses element's font size.

Reply:
You're right, it was related to a followup pr but irrelevant here 👍